firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/model/CrashlyticsReport.java

```diff
@@ -185,7 +188,7 @@ public CrashlyticsReport withSessionEndFields(
   /** Augment an existing {@link CrashlyticsReport} with the given app quality session id. */
   @NonNull
   public CrashlyticsReport withAppQualitySessionId(@Nullable String appQualitySessionId) {
-    Builder builder = toBuilder();
+    Builder builder = toBuilder().setAppQualitySessionId(appQualitySessionId);
```

Comment:
Do we want to put down some notes around why we set AQS ID for two different place?